src/pages/Meeting/DetailMeatingManagePage.tsx

```
@@ -0,0 +1,200 @@
+          {/* 오른쪽: 참여자 */}
+          <div className="flex flex-col flex-1">
+            <h2 className="text-[#2C2C2C] text-[18px] font-medium mb-5">
+              토론 참여자
+            </h2>
+
+            <div className="bg-white border-[2px] border-[#EAE5E2] rounded-[12px] p-6 min-h-[540px]" >
+              <div className="space-y-4">
+                {participants.map((member: ClubMember) => (
+                  <div className="flex items-center justify-between border-b border-[#EAE5E2] pb-3">
+                    <div className="flex items-center gap-4 min-w-[180px]">
+                      <img src={member.basicInfo.profileImageUrl} className="w-[36px] h-[36px] rounded-full bg-[#DADADA]" />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)